packages/@okta/vuepress-site/docs/guides/add-an-external-idp/create-authz-url/index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Create the Authorization URL
+---
+The Okta Identity Provider that you created in the <GuideLink link="../configure-idp-in-okta">second step</GuideLink> generated an authorize URL with a number of blank parameters that you must now fill in. The authorize URL initiates the authorization flow that authenticates the user with the Identity Provider.
+
+In the URL, replace {your OktaDomain} with your org's base URL, and then replace the following values:
+
+* `client_id`: Use the `client_id` value that you obtained from the OpenID Connect client application in the <GuideLink link="../create-app-in-okta">previous section</GuideLink>. This is not the `client_id` from the Identity Provider.
+
+* `scope`: Determines the claims that are returned in the ID token. Include the scopes that you want to request authorization for and separate each by a space. You need to include at least the `openid` scope. You can request any of the standard OpenID Connect scopes about users, such as `profile` and `email` as well as any custom scopes specific to your Identity Provider.
+
+* `response_type`: Determines which flow is used. This should be `id_token`.
+
+* `response_mode`: Determines how the authorization response should be returned. This should be `fragment`.
+
+* `state`: Protects against cross-site request forgery (CSRF). Can be any value.
+
+* `nonce`: A string included in the returned ID token. Use it to associate a client session with an ID token and to mitigate replay attacks. Can be any value.
+
+* `redirect_uri`: The location where Okta returns a browser after the user finishes authenticating with their identity provider. This URL must start with HTTPS and must match one of the redirect URIs that you configured in the <GuideLink link="../create-app-in-okta">previous section</GuideLink>.
```

Comment:
Do we have a rationale for the order in which parameters are listed?

@ZoeMonosson-okta identified that this order does not match the example URL generated by the Developer Console (or Admin Console). The console uses an order that is the same as the `example of a complete URL` below. This makes it difficult to use this list, as the developer can't just go down it one by one when constructing their Authorize URL.

A quick fix might just be updating this list to the same order as the example / console.

Other rationales I could imagine:
- Roughly mirroring an authorization flow chronology: first, which client is the target; then, what kind of request is this; at the end, where should the client be redirected to, etc.
- Alphabetical

(I assume that when processing the request, Okta will accept the parameters in any order.)

Reply:
@alexdahl-okta - I think the order matching the example URL generated by the Developer Console makes sense. I'm afraid that I missed @ZoeMonosson-okta identifying that in the Google Doc that she shared with me. Is there an additional place that I need to be looking for comments from Zoe? Thank you!

Reply:
Oh nowhere additional, I was just talking with her earlier today and these things came up – probably not flagged previously! Thanks for tracking everything!